gcloud/logging/logger.py

```diff
@@ -89,10 +113,15 @@ def log_text(self, text, client=None):
                 },
             }],
         }
+
+        labels = self._get_labels(labels)
+        if labels is not None:
+            data['entries'][0]['labels'] = labels
```

Comment:
That commit factors out all the "make a log entry resource" logic into a helper method, leaving the public ones much slimmer (they don't have to "reach down" into the list of entries).